firestore-translate-text/functions/src/index.ts

```diff
@@ -79,13 +82,78 @@ export const fstranslate = functions.handler.firestore.document.onWrite(
     } catch (err) {
       logs.error(err);
     }
-  }
-);
+  });
+
+export const fstranslatebackfill = functions.tasks
+  .taskQueue()
+  .onDispatch(async (data: any) => {
+    const runtime = getExtensions().runtime();
+    if (!config.doBackfill) {
+      await runtime.setProcessingState(
+        "PROCESSING_COMPLETE",
+        "Existing documents were not translated because backfill was not requested."
```

Comment:
We can add "If you want to translate existing documents, reconfigure your extension and choose 'yes' for 'Translate existing documents?' Only missing translations will be requested." or something like that.